crates/fuel-core/src/executor.rs

```diff
@@ -1699,6 +1699,7 @@ where
                     db.update_tx_status(
                         id,
                         TransactionStatus::Success {
+                            tx_id: *id,
```

Comment:
We don't need to store `tx_id` in the database because we can fetch it from the key. The GraphQL type may have this field, while the type that we use across `fuel-core` does not.